subprojects/files/src/test/groovy/org/gradle/api/internal/file/collections/GeneratedSingletonFileTreeTest.java

```diff
@@ -29,67 +28,24 @@
 import java.io.File;
 import java.io.IOException;
 import java.io.OutputStream;
-import java.util.List;
+import java.util.Collections;
 import java.util.concurrent.atomic.AtomicInteger;
 
-import static org.gradle.api.file.FileVisitorUtil.assertCanStopVisiting;
 import static org.gradle.api.file.FileVisitorUtil.assertVisits;
 import static org.gradle.api.internal.file.TestFiles.directoryFileTreeFactory;
-import static org.gradle.api.tasks.AntBuilderAwareUtil.assertSetContainsForAllTypes;
 import static org.gradle.util.WrapUtil.toList;
 import static org.hamcrest.Matchers.equalTo;
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.*;
+
+public class GeneratedSingletonFileTreeTest {
 
-public class MapFileTreeTest {
     @Rule
     public final TestNameTestDirectoryProvider tmpDir = new TestNameTestDirectoryProvider();
     private TestFile rootDir = tmpDir.getTestDirectory();
-    private MapFileTree tree;
 
     @Before
     public void setup() {
         NativeServicesTestFixture.initialize();
-        tree = new MapFileTree(rootDir, TestFiles.fileSystem(), directoryFileTreeFactory());
-    }
-
-    @Test
-    public void isEmptyByDefault() {
-        List<String> emptyList = toList();
-        assertVisits(tree, emptyList, emptyList);
-        assertSetContainsForAllTypes(tree, emptyList);
-    }
-
-    @Test
-    public void canAddAnElementUsingAClosureToGeneratedContent() {
```

Comment:
We still would need a test that we can use closures for generating content, right?
Or probably not, since all the code is internal and we only support Actions...

Reply:
I think this can be removed. Also it wasn't clear to me how that test covered that in the 1st place